ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+## Description
+[Please provide a general introduction to the issue/proposal.]
+
+[If reporting a bug, attach the entire traceback from Python and follow the way to reproduce below]
+[If proposing an enhancement/new feature, provide links to related articles, reference examples, etc.]
+
+
+## Way to reproduce
+[If reporting a bug, please include the following important information:]
+- [ ] Code example
+- [ ] Relevant images (if any)
+- [ ] Operating system and version (run `python -c "import platform; print(platform.platform())"`)
+- [ ] Python version (run `python -c "import sys; print("Python", sys.version)"`)
+- [ ] dipy version (run `python -c "import dipy; print(dipy.__version__)"`)
+- [ ] dependency version (numpy, scipy, vtk)
```

Comment:
Do we want to know versions of other dependencies or just these three?

Reply:
I suppose they are the most important, but I add the others dependencies to get the complete information ([last commit](https://github.com/nipy/dipy/pull/1361/commits/0c2dc2ae7f1a463503f83613e04ce777ef05a542))